automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala

```diff
@@ -12,6 +13,9 @@ trait Clickable { this: Component =>
     * Click on the element modeled by this Component
     */
   def doClick()(implicit webDriver: WebDriver): Unit = {
+    // before click, wait up to 10 seconds for element become clickable
```

Comment:
Good find! This definitely needs to be fixed (though I'm not 100% convinced that this is what was happening with this test... though it still might help... let's chat).

If you dig into the implementation of `elementToBeClickable`, it's based on the enabled state which we already have support for in `WebBrowserUtil`. I suggest the following implementation:

```
  def doClick()(implicit webDriver: WebDriver): Unit = {
    awaitVisible()
    scrollToVisible()
    click on (await enabled query)
  }
```

Optionally, you can add this function to `Component`:

```
  def awaitEnabled(): Unit = await enabled query
```

and use this implementation:

```
  def doClick()(implicit webDriver: WebDriver): Unit = {
    awaitVisible()
    scrollToVisible()
    awaitEnabled()
    click on query
  }
```